changes/bug25568

```diff
@@ -0,0 +1,4 @@
+  o Minor bugfixes (onion services):
```

Comment:
This commit's message has a typo, it should say "Bug"

Reply:
Fixed it.